sci-libs/vtk/vtk-9.3.1.ebuild

```diff
@@ -388,6 +392,35 @@ src_configure() {
 		-DVTK_WRAP_PYTHON="$(usex python)"
 	)
 
+	if use minimal; then
```

Comment:
This is a bit confusing to me, `use minimal` is enabling a lot of extra options. Naively I would expect this the other way around, i.e. `use minimal` disabling a bunch of options.